kafka-ui-react-app/src/components/App.tsx

```diff
@@ -38,16 +38,18 @@ const queryClient = new QueryClient({
 });
 
 const App: React.FC = () => {
+  const [isDarkMode, setDarkMode] = React.useState<boolean>(false);
+
   return (
     <QueryClientProvider client={queryClient}>
       <GlobalSettingsProvider>
-        <ThemeProvider theme={theme}>
+        <ThemeProvider theme={isDarkMode ? darkTheme : theme}>
```

Comment:
Am I dreaming? true/false for theming rework of this scale? I had hopes for you guys this afternoon. Hope you will iterate past this soon. A simple palette would do, not even CSS variables.